deployments/systemd/packages/Makefile

```diff
@@ -25,17 +25,20 @@ REGISTRY ?= nvcr.io/nvidia
 IMAGE := $(REGISTRY)/$(NAME)
 endif
 
+# strip 'v' from version string
+PACKAGE_VERSION := $(VERSION:v%=%)
```

Comment:
Out of scope for this PR: in the libnvidia-container and nvidia-container-toolkit packages, I started generating the changelog automatically as part of the build to streamline things. Not sure if that's something we want to try here.